google/cloud/forseti/common/gcp_type/firewall_rule.py

```diff
@@ -66,6 +66,7 @@ def __init__(self, validate=False, **kwargs):
         self.resource_id = kwargs.get('id')
         self.create_time = kwargs.get('firewall_rule_create_time')
         self.name = kwargs.get('firewall_rule_name')
+        self.hierarchical_name = kwargs.get('firewall_rule_hierarchical_name')
```

Comment:
This should probably default to a string, as get_resource_ancestors is causing the tests to fail due to the rsplit on a None hierarchical_name.

Reply:
Done